.markdownlint.json

```diff
@@ -5,5 +5,6 @@
     },
     "MD033": {
         "allowed_elements": ["a", "img", "p"]
-    }
+    },
+    "MD013": false
```

Comment:
I know it's annoying but I think we do want to leave this enabled, it makes markdown a lot more readable.

Reply:
I needed it to add the documentation at the root here https://github.com/neo4j/graphql/pull/954/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R44. Personally I think it's too strict but couldn't find a way to increase the numbers.

Reply:
I agree, 80 is too short. 😞  Let's leave it disabled for now, but keep it in mind to look at. 🙂